composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "broadway/read-model-mongodb",
+    "description": "A MongoDB ReadModel implementation for Broadway.",
+    "license": "MIT",
+    "homepage" : "https://github.com/broadway/read-model-mongodb",
+    "authors": [
+        {
+            "name": "Robin van der Vleuten",
+            "email": "robin@webstronauts.co"
+        }
+    ],
+    "require": {
+        "php": ">=7.0",
+        "broadway/broadway": "^1.0",
```

Comment:
ext-mongodb: * ? or is it already required by mongodb/mongodb?

Reply:
It is already required by the mongodb package.